test/CMakeLists.txt

```diff
@@ -111,7 +111,10 @@ macro(onedpl_add_test test_source_file switch_off_checked_iterators)
     set_tests_properties(${_test_name} PROPERTIES SKIP_RETURN_CODE 77)
 
     if (DEFINED DEVICE_SELECTION_LINE)
-        set_tests_properties(${_test_name} PROPERTIES ENVIRONMENT ${DEVICE_SELECTION_LINE})
+        file(GLOB_RECURSE DS_TESTS "parallel_api/dynamic_selection/*.cpp")
+        if (NOT "${DS_TESTS}" MATCHES "${_test_name}") # do not set SYCL_DEVICE_FILTER for device selector tests
```

Comment:
Should this comment read "do not set SYCL_DEVICE_FILTER for dynamic selection tests"?

Reply:
Fixed